shell/platform/fuchsia/flutter/tests/integration/embedder/child-view/lib/child_view.dart

```diff
@@ -46,7 +50,7 @@ class TestApp {
 
   void pointerDataPacket(PointerDataPacket packet) {
     for (final data in packet.data) {
-      if (data.change == PointerChange.up) {
+      if (data.change == PointerChange.down) {
```

Comment:
intentional?

Reply:
Yes, the in-tree copy of this test is listening for down.  This was the source of a test failure here, since fuchsia is not sending up events correctly ATM.  For now I changed it to down to match the in-tree test, and @abrush21 is going to fix the up bug and add a test:  see https://fuchsia-review.googlesource.com/c/fuchsia/+/710568

Reply:
sg thanks